UnitsNet.Tests/CustomCode/CoefficientOfThermalExpansionTests.cs

```diff
@@ -35,6 +35,12 @@ public class CoefficientOfThermalExpansionTests : CoefficientOfThermalExpansionT
 
         protected override double InverseKelvinInOneInverseKelvin => 1.0;
 
+        protected override double PartsPerMillionPerDegreeCelsiusInOneInverseKelvin => 1e6;
```

Comment:
I don't know what naming conventions are more widely used in this domain, but it seems to me from googling that "inverse" is maybe non-standard? 

Instead, I see PerKelvin, PerDegreeCelsius and PartsPerMillionDegreeCelsius in places like this source:

https://www.sensorsone.com/temperature-coefficient-converter/
![image](https://github.com/angularsen/UnitsNet/assets/787816/5d68120c-3f0c-4e94-8a8b-a921d55a65b6)

Should we rename the Inverse units or are they fine as-is?
It would require adding new units and obsoleting the old ones, until the next major version bump.

Reply:
It also seems like we could maybe abbreviate to `PpmPerDegreeCelsius` to keep it a bit shorter, while still fairly clear what it means. Thoughts?

Reply:
Yes agree on a ppm prefix, do you prefer that capitalisation rather than say `PPMPerDegreeCelsius`?

Also prefer the `Per...` over `Inverse...`, as far as I can tell, changing this would involve:
- Adding the three new `Per...` units, with same definitions as existing units. Adding unit tests definitions as required etc.
- Adding "ObsoleteText: " to the `Inverse...` unit definitions in json.
- Change the base units from `InverseKelvin` to `PerKelvin`, or should this be left until the next major change?

Reply:
`Ppm` would follow .NET naming conventions for abbreviations in my experiencee.

Yes, your approach looks exactly right.

I'm trying to think of scenarios for when changing base units breaks anything, but I can't really think of any. It is primarily used in conversion functions to convert via, but since they are equal that requires no change. 

All constructors and factory methods specify a unit.
Serialization specifies the unit.

One minor change would be that `Zero` property would have a different unit, but again, I don't think this will be a breaking change for most.

I say let's just change it 👍